docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Headless Login (Preview)
+description: Headless Login
+---
+
+## Introduction
+
+<Admonition type="warning" title="Preview">
+  Headless login is currently in Preview mode.
+</Admonition>
+
+Headless login provides Teleport users with a secure way to authenticate with WebAuthn from a remote machine. This enables use cases which usually are not compatible with WebAuthn features, such as:
+
+- Logging into Teleport with WebAuthn from a remote dev box
+- Connecting to a Teleport Node from a remote dev box with Per-session MFA 
+- Performing `tsh scp` from one Teleport Node to another with Per-session MFA
+- Logging into Teleport on a machine without a WebAuthn compatible browser
+
+<Admonition type="note" title="Headless Login Support">
+  Headless login only supports following `tsh` commands:
+  
+  - `tsh ls`
+  - `tsh ssh`
+  - `tsh scp`
+
+  In the future, headless login may be extended to other `tsh` commands.
+</Admonition>
+
+## Prerequisites
+
+- A Teleport cluster with WebAuthn configured.
+  See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
+- WebAuthn hardware device, such as YubiKey or SoloKey.
+- A Web browser with [WebAuthn support](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/).
+
+A Teleport cluster capable of WebAuthn is automatically capable of Headless login.
+
+## Step 1/2. Initiate headless login
+
+Run a headless tsh command with the `--headless` flag. This will initiate headless login, printing a URL and `tsh` command.
+
+```code
+$ tsh ls --headless --proxy=example --user=user
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/86172f78-af7c-5935-a7c1-ed06b94f17dc
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=dev --proxy=proxy.example.com:3080 86172f78-af7c-5935-a7c1-ed06b94f17dc
+```
+
+## Step 2/2. Approve headless Login
+
+To approve the headless login, go to the URL in your local web browser or enter the `tsh` command in your local terminal. You will be prompted to approve the login with WebAuthn verification. Once approved, your initial `tsh --headless <command>` should continue as if you had logged in locally.
```

Comment:
I'd explicitly mention that the URL is printed by `tsh` and needs to be copied and pasted in our browser. I'd also double-check which browser do we support.

Reply:
"A Web browser with WebAuthn support" is one of the prerequisites.